test-result-summary-client/src/TestCompare/TestInfo.jsx

```diff
@@ -5,20 +5,15 @@ const FormItem = Form.Item;
 
 export default class TestInfo extends Component {
     render() {
-        const { data: { url, buildName, buildNum, testName }, onChange } = this.props;
+        const { data: { compareType, buildUrl, testName }, onChange } = this.props;
+
         return <Form className="ant-advanced-search-form" onSubmit={this.handleSearch}>
-            <Row gutter={40}>
-                <FormItem>
-                    <Input prefix={<Icon type="user" style={{ fontSize: 13 }} />} placeholder="Jenkin Server URL" value={url} onChange={onChange.bind( null, 'url' )} />
-                </FormItem>
-                <FormItem>
-                    <Input prefix={<Icon type="user" style={{ fontSize: 13 }} />} placeholder="Build Name" value={buildName} onChange={onChange.bind( null, 'buildName' )} />
-                </FormItem>
-                <FormItem>
-                    <Input prefix={<Icon type="user" style={{ fontSize: 13 }} />} placeholder="Build Number" value={buildNum} onChange={onChange.bind( null, 'buildNum' )} />
+            <Row>
+                <FormItem label={compareType + " Build URL"}>
+                    <Input prefix={<Icon type="user" style={{ fontSize: 13 }} />} placeholder="Jenkins Build URL" value={buildUrl} onChange={onChange.bind( null, 'buildUrl' )} />
                 </FormItem>
-                <FormItem>
-                    <Input prefix={<Icon type="user" style={{ fontSize: 13 }} />} placeholder="Test Name" value={testName} onChange={onChange.bind( null, 'testName' )} />
+                <FormItem label={compareType + " Test Name (Optional)"}>

```

Comment:
Test Name is not optional. It is required for the test build.

Reply:
"Optional" has been removed from form label, placeholder, commit message, and PR info. New test Compare is as following:
![image](https://user-images.githubusercontent.com/48260423/82587825-db376980-9b67-11ea-8f89-14ea23384857.png)